### DISCUSSION TITLE
[Transportation] Accessing the subclass and subclass_rules properties of segments

### DISCUSSION THREAD

**Discussion by gregorhd** [OvertureMaps/data#207] [Discussion]
### Category Feedback

Hi there,

looking at the `segment` [schema](https://docs.overturemaps.org/schema/reference/transportation/segment/), I'm intrigued by the `subclass` and `subclass_rules` properties. How can they be queried in Athena? There obviously aren't columns by that name in the latest release `v2024_07_22_0`.  It also doesn't seem like these properties are elements in an array somewhere.

What am I missing? Thanks!

### Dependency with other categories, if any.

Schema

**Comment by skmoore** [comment]
The subclass and subclass_rules properties are new in the August release, more details here: https://docs.overturemaps.org/release/latest/#theme-specific-updates

**Reply by gregorhd** [reply]
Ah d'oh! Now I can see the new release `v2024_08_20_0` in Athena as well. Works like charm. Cheers!
